liability.qmd

```diff
@@ -8,12 +8,60 @@ title: "Liability with OS"
+
+Given the potential risks associated with software use, such as system malfunctions or data breaches, the liability and warranty of the source code provider become critical considerations. 
+
+Most open-source licenses include a disclaimer to shield developers from claims. However, the responsibility for liability and warranty doesn't solely rest with open-source users. It extends to all software users, including those using commercial software.
+
+Andy Nicholls has helpfully provided a link to the [EMA's Notice on validation and qualification of software tools used in clinical trials](https://www.ema.europa.eu/en/documents/regulatory-procedural-guideline/notice-sponsors-validation-and-qualification-computerised-systems-used-clinical-trials_en.pdf){target="_blank"}. This document clearly states that the sponsor bears the ultimate responsibility for the validation of the computerised system. They must provide adequate documented evidence of the validation process. 
+
+While the sponsor can rely on qualification documentation provided by the vendor, they must assess the adequacy of the vendor's qualification activities. Based on a documented risk assessment, the sponsor may also need to perform additional qualification and validation activities.
+
+>The sponsor is ultimately responsible for the validation of the computerised system and for providing adequate documented evidence on the validation process. [...] The sponsor may rely on qualification documentation provided by the vendor, if the qualification activities performed by the vendor have been assessed as adequate. However, the sponsor may also have to perform additional qualification (and validation) activities based on a documented risk assessment.
+
+**Summary**: Regardless of whether open-source or commercial software is used, the sponsor carries the responsibility for warranty and liability. While commercial software requires reliance on the vendor, the transparency of open-source software offers additional avenues for assurance and accountability.
+
+## Liability and Warrenty according Open Source Licenses
+
+Most open source licenses include a disclaimer of liability and warranty. So according the license there is no liability and warrenty from the source code provider.
+
+Examples:
+
+-   [MIT License](https://opensource.org/licenses/MIT){target="_blank"}
+
+    >The software is provided “as is”, **without warranty** of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement. In **no event shall** the authors or copyright holders **be liable** for any claim, damages or other liability, whether in an action of contract, tort or otherwise, arising from, out of or in connection with the software or the use or other dealings in the software.
+
+-   [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0.html){target="_blank"}
+
+    >7. Disclaimer of Warranty <br/>
```

Comment:
In the Visual version of this qmd, a lot of these "<br>" and "<br/>" html tags are being rendered, instead of suppressed. I don't think they are needed, since quarto can put in line breaks. There are a number of them from line 41 onward.

Reply:
The {target="_blank"} tags are not visible in the rendition when rendered with quarto (at least from command line - outside RStudio). These are not visible in the rendition:
![image](https://github.com/phuse-org/OSTCDA/assets/55694577/4871a75c-3fba-42d6-976b-95d9dd37739a)

These are needed so the links opens in a new window. I will keep them and check the final rendition in GitHub.